server.js

```diff
@@ -2,11 +2,13 @@
 
 'use strict';
 
-const Hapi = require('hapi');
+const Hapi = require('hapi'),
+Db = require('./lib/database');
 
 // Create a server with a host and port
 const server = new Hapi.Server();
 server.connection({
+    host: 'localhost',
```

Comment:
Yeah, I took this out when I was being a bit touchy with trying to get it to work on Heroku.  I somehow got the idea that this was causing the port binding to only work for localhost.  I imagine I broke local node stuff so it is good you put it back.